src/conn/conn_sweep.c

```diff
@@ -292,8 +292,13 @@ __sweep_server(void *arg)
 	__wt_seconds(session, &last);
 	for (;;) {
 		/* Wait until the next event. */
-		__wt_cond_wait(session, conn->sweep_cond,
-		    min_sleep * WT_MILLION, __sweep_server_run_chk);
+		if (FLD_ISSET(conn->timing_stress_flags,
+			WT_TIMING_STRESS_AGGRESSIVE_SWEEP))
+			__wt_cond_wait(session, conn->sweep_cond,
+				min_sleep * 10000, __sweep_server_run_chk);
```

Comment:
Same comment about indentation as above. Also, it is easier to read `10 * WT_THOUSAND` as compared to `10000`.